Falcon/Falcon/Rendering/PipeLine/Renderer.cpp

```diff
@@ -86,14 +92,15 @@ void Renderer::CreateDrawStates()
 /**
 *Function to Set the relevant data in the draw states.
 */
-void Renderer::SetDrawStates(boost::container::vector<Entity*, fmemory::StackSTLAllocator<Entity*>>* entities)
+void Renderer::SetDrawStates(boost::container::vector<Entity*, fmemory::StackSTLAllocator<Entity*>>* entities, glm::mat4 projection)
```

Comment:
Use ref instead of copy?

Reply:
not possible. The projection being passed is defined on the stack.